opm/grid/LookUpData.hh

```diff
@@ -34,11 +34,18 @@
 
 #include <dune/grid/common/mcmgmapper.hh>
 
+#include <opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp>
 #include <opm/grid/cpgrid/Entity.hpp>
 
+#include <string>
 #include <type_traits>
 #include <vector>
 
+namespace Opm
+{
+class FieldPropsManager;
+}
+
```

Comment:
No, we don't. It was there before I realized we need the header since we are invoking methods of that class. I remove Entity forward declaration too, for the same reason. Thanks!